src/components/pages/Authentication/hooks.ts

```diff
@@ -5,32 +5,61 @@ import { useLocation, useNavigate } from "react-router-dom";
 import { useAuth, useCleanUrl } from "@reearth/auth";
 import { useGetTeamsQuery } from "@reearth/gql";
 import { useT } from "@reearth/i18n";
-import { useTeam, useNotification } from "@reearth/state";
+import { useTeam, useNotification, useUserId } from "@reearth/state";
 
 // TODO: move hooks to molecules (page components should be thin)
 export default () => {
   const { isAuthenticated, isLoading, error: authError, logout, login } = useAuth();
   const [error] = useCleanUrl();
   const navigate = useNavigate();
   const location = useLocation();
-  const [currentTeam, setTeam] = useTeam();
+  const [currentWorkspace, setCurrentWorkspace] = useTeam();
+  const [currentUserId, setCurrentUserId] = useUserId();
   const [, setNotification] = useNotification();
   const passwordPolicy = window.REEARTH_CONFIG?.passwordPolicy;
   const t = useT();
 
   const { data, loading } = useGetTeamsQuery({ skip: !isAuthenticated });
-  const teamId = currentTeam?.id || data?.me?.myTeam.id;
+
+  if (isAuthenticated && !currentUserId) {
+    setCurrentUserId(data?.me?.id);
+  }
```

Comment:
Can you be sure on first render that we will have values for `isAuthenticated` and `currentUserId`? If so, this is fine, but if not this should be in a useEffect so that WHEN we get those values we will check and setCurrentUserId if appropriate

Reply:
I don't want this to be inside useeffect as it will be set for one time for each user on his browser cause the currentUser Id is also a atomWithLocalStorage and I don't want this to change once onther user use same browser